### PR TITLE
Update language feature status for interpolated string improvements.

### PR DESCRIPTION
**Pull request by 333fred**

### CODE REVIEW

**Comment by 333fred** [comment]
@dotnet/roslyn-compiler for a quick review.

**Comment by 333fred** [comment]
(Note the branch doesn't exist yet, it will shortly)

**Review by cston — DISMISSED** [review]

**Comment by 333fred** [comment]
Highly recommend using the rich diff, as I removed all the extraneous spaces. Unless you have an ultrawide monitor and shrink the text you're not reading the raw markdown table anyway (ask me how I know).

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -8,26 +8,29 @@ efforts behind them.
...
+| Feature | Branch | State | Developers | Reviewer | LDM Champ |
+| ------- | ------ | ----- | ---------- | -------- | --------- |
+| [Record structs](https://github.com/dotnet/csharplang/issues/4334) | [record-structs](https://github.com/dotnet/roslyn/tree/features/record-structs) | [In Progress](https://github.com/dotnet/roslyn/issues/51199) | [jcouv](https://github.com/jcouv) | [AlekseyTs](https://github.com/AlekseyTs), [RikkiGibson](https://github.com/RikkiGibson) | [jcouv](https://github.com/jcouv) |
+| [Global Using Directive](https://github.com/dotnet/csharplang/issues/3428) | [GlobalUsingDirective](https://github.com/dotnet/roslyn/tree/features/GlobalUsingDirective) | [In Progress](https://github.com/dotnet/roslyn/issues/51307) | [AlekseyTs](https://github.com/AlekseyTs) | [333fred](https://github.com/333fred), [cston](https://github.com/cston) | [AlekseyTs](https://github.com/AlekseyTs) |
+| [File-scoped namespace](https://github.com/dotnet/csharplang/issues/137) | TBD | In Progress | [chsienki](https://github.com/chsienki) | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson) | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) |
```
Comment (chsienki) [inline docs/Language Feature Status.md:15]:
```suggestion
| [File-scoped namespace](https://github.com/dotnet/csharplang/issues/137) | [FileScopedNamespaces] (https://github.com/dotnet/roslyn/tree/features/FileScopedNamespaces) | In Progress | [chsienki](https://github.com/chsienki) | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson) | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) |
```

**Comment by RikkiGibson** [comment]
I honestly have considered changing this table to just straight HTML instead of really long markdown lines that can't be broken up. But that might inspire a different objection from my teammates.

**Review by chsienki — APPROVED** [review]
Suggested updating the file scoped namespaces too, rather than submitting a second PR ;)

**Comment by 333fred** [comment]
> Suggested updating the file scoped namespaces too, rather than submitting a second PR ;)

Now I feel bad, because I just noticed that you had a branch and updated it on my own, and I could have just hit "commit suggestion"

**Pull request merged**